.gitignore

```diff
@@ -9,3 +9,6 @@ docs/_build
 .testrepository/
 .tox
 venv/
+.mypy_cache
+.vscode
+
```

Comment:
These are local files, which should be managed by a global gitconfig instead.

Reply:
Thanks, fixed